### PR TITLE
Ers 242 activity system test

### PR DESCRIPTION
**Pull request by robertkrantz**

### CODE REVIEW

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

File: unionvms-test/src/test/java/eu/europa/ec/fisheries/uvms/docker/validation/system/ModuleAvailableSystemIT.java [hunk 1]
```diff
@@ -104,6 +105,7 @@ public void checkReportingAccessTest() {
 	}
 
 	@Test
+	@Ignore
```
Comment (RobinMilding) [inline unionvms-test/src/test/java/eu/europa/ec/fisheries/uvms/docker/validation/system/ModuleAvailableSystemIT.java:108]: Should have a comment about why the test is ignored

**Review by RobinMilding — APPROVED** [review]

**Comment by robertkrantz** [comment]
test me

**Comment by robertkrantz** [comment]
test me

**Pull request merged**